pkg/lib/serverTime.js

```diff
@@ -193,7 +193,7 @@ export function ServerTime() {
                             .catch(e => {
                                 ntp_waiting_resolve();
                                 ntp_waiting_resolve = null;
-                                console.error(e.message);
+                                console.error("Failed to call SetNTP:", e.message);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19511-20231020-065404-313488a1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)